src/h5web/visualizations/heatmap/HeatmapToolbar.tsx

```diff
@@ -22,7 +24,8 @@ function HeatmapToolbar(): ReactElement {
     <Toolbar>
       <DomainSlider />
       <Separator />
-      <ColorMapSelector />
+
+      <ColorMapSelector value={colorMap} onChange={setColorMap} />
```

Comment:
The selector is now controlled for consistency with the other toolbar components (the domain slider will be next).